var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -19,11 +19,10 @@ class Zlib(Package):
     # URL must remain http:// so Spack can bootstrap curl
     url = "https://zlib.net/fossils/zlib-1.2.11.tar.gz"
```

Comment:
I just saw this comment and this URL is not `http://` anymore

Reply:
That's probably best discussed in a new issue.